Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs

```diff
@@ -28,6 +32,21 @@ public AuthServiceTester()
             _output = new TestOutputHelper();
         }
 
+        [Theory]
+        [InlineData(11, 11, true)]
+        [InlineData(2, 1, false)]
+        public void GenerateAccessTokenTest(int id1, int id2, bool expected)
```

Comment:
In the method name, include what you are testing for.

```suggestion
        public void GenerateAccessToken_TestDifferentEmployeesGetDifferenTokens(int id1, int id2, bool expected)
```